test/gateway/eip20_cogateway/redeem.js

```diff
@@ -28,7 +28,7 @@ const MockToken = artifacts.require('MockToken');
 
 const { MessageStatusEnum } = messageBus;
 
-const nonce = new BN(1);
+const nonce = new BN(0);
```

Comment:
why hardcode it and not get it from the gateway?

Reply:
Why not hardcoded? 

We already know that the nonce will start at 0. 
This will also assert that nonce starts at 0.

Reply:
ok